src/main.cpp

```diff
@@ -2549,6 +2549,22 @@ gb_internal int strip_semicolons(Parser *parser) {
 
 gb_internal void init_terminal(void) {
 	build_context.has_ansi_terminal_colours = false;
+
+    gbAllocator a = heap_allocator();
+
+    char const *no_color = gb_get_env("NO_COLOR", a);
```

Comment:
This latest patch looks great, however did you use 4 spaces instead of tabs to indent? It doesn't align as you can see.
![image](https://github.com/odin-lang/Odin/assets/194614/61af2501-b18d-448b-86fe-74927454faa5)

Reply:
Damnit, let me fix that